### DISCUSSION TITLE
Telegram Thread_id

### DISCUSSION THREAD

**Discussion by DAVIDMARTINEZROBLES** [getwud/wud#506] [Ideas]
Hi.

Its possible add the posibility to add option to send the alert to specific thread 

   environment:
      - WUD_AUTH_BASIC_JOHN_USER=admin
      - WUD_AUTH_BASIC_JOHN_HASH=xxxxxxxxxxx
      - WUD_TRIGGER_SMTP_GMAIL_SIMPLETITLE=CONTAINER $${name} CAN BE UPDATED
      - WUD_TRIGGER_SMTP_GMAIL_SIMPLEBODY=$${name} update |$${local}| ----> |$${remote}|
      - WUD_TRIGGER_TELEGRAM_1_BOTTOKEN=**xxxxxxxxxxxxxxxxxxx**
      - WUD_TRIGGER_TELEGRAM_1_CHATID=**xxxxxxxxxxxxxxxxxx**
      - WUD_TRIGGER_TELEGRAM_1_THREAD_ID=**XXX**
 
or something like this
WUD_TRIGGER_TELEGRAM_1_CHATID=**-XXXXXXXXX**&message_thread_id=**XXX**

**Comment by darkymehdi** [comment]
would love that too :) thanks

**Comment by fmartinou** [comment]
Hi,

It's now implemented (https://github.com/getwud/wud/pull/1240).
It will be available in the next release.
